src/sentry/issues/grouptype.py

```diff
@@ -116,6 +117,7 @@ class GroupType:
     description: str
     category: int
     noise_config: Optional[NoiseConfig] = None
+    default_priority: int = PriorityLevel.MEDIUM
```

Comment:
what happens if there is Celery worker using this dataclass that has old code, but is invoked from the new code, or other way around? Will it work just fine?

Reply:
hmm good question, thinking "aloud"
- old code, new schema should be fine since it wouldn't use priority anywhere
- new code, old schema would be an issue but I think splitting the GroupType schema change out into a separate PR would fix it? WDYT?

Reply:
cc @wedamija should we split this PR? I'm not clear on the risk as it deploys

Reply:
@vartec What's the concern you have here? We don't pickle classes between processes, so code changes shouldn't be a problem. The main thing we need to be careful of is the data schema changes - so if an old process without these changes gets a value with priority will anything break? And if the new code gets data without priority will it break?

Reply:
I think it would be safer to split up this PR and add the GroupType.priority defaults first. Once that's in prod, the next change can safely make use of the field. Splitting this up now.